src/EventListener/DoctrineTaskExecutionListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of php-task library.
+ *
+ * (c) php-task
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Task\TaskBundle\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Task\Event\TaskExecutionEvent;
+
+/**
+ * Listens on task-execution events.
+ */
+class DoctrineTaskExecutionListener
```

Comment:
Ok, that's true, but I think it is a little bit hidden in the configuration... And since this class is already bound to an event I would also not fear implementing the `EventSubscriberInterface`... But well, I think that's just personal preference. It's also ok for me this way.